src/css/override_vars.scss

```diff
@@ -1,4 +1,8 @@
 
+// Path to override for web design standards
+$static-img-path: '../img' !default;
+$static-font-path: '../fonts' !default;
```

Comment:
Because it's the overrides to the standards, it's `fonts` because the standards is `fonts`.

It's confusing because I need to be able to override both the standards and cg-style paths.